torch/_inductor/codecache.py

```diff
@@ -2051,8 +2051,12 @@ def _pad_to_alignment(raw_bytes: bytes) -> bytes:
                 else:
                     run_command_and_check(link_cmd)
 
-                for o_file in [output_o, consts_o]:
-                    # remove .o files to save disk space since we already have the .so file
+                for o_file in [
+                    output_o,
+                    consts_o,
+                    os.path.splitext(consts_o)[0] + ".S",
```

Comment:
what's a .S? 😮

Reply:
copied from the weights emitting logic from the macos path. Either .s or .S should be fine for weights emitting purpose.